test.properties

```diff
@@ -5290,7 +6204,7 @@
         modules-integration-mysql57-jdk8
 
     test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][search-remote]=\
-        ${test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][search]} AND \
+        (${test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][search]}) AND \
```

Comment:
@joshchong from the history I guess you can help us.
Can we add parenthesis `()` here?
This case starts with `${` and ends with `}`

Reply:
sure that's fine

Reply:
Thanks.